content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +162,45 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+For every `dvc.yaml` file, a corresponding `dvc.lock` (YAML) file is created or
+updated by DVC as well. Its purpose is to lock the exact contents of all files
+involved in the pipeline(s) at the time that the pipeline was last committed
+(after using `dvc add`, `dvc run`, `dvc repro`, or `dvc commit`, for example).
+
+This means that regular <abbr>dependencies</abbr> and all <abbr>outputs</abbr>
+(including [metrics](/doc/command-reference/metrics) and
+[plots](/doc/command-reference/plots) files) are also listed in `dvc.lock`, but
+with additional fields that store the `md5` hash value (or `etag` for HTTP
```

Comment:
let's just use hash, w/o going to deep (like etag for HTTP, etc), especially since you include an example

Reply:
Hmm per https://github.com/iterative/dvc.org/pull/1448#discussion_r443124777 this is already a new bullet list actually. Isn't this the document where we want to have the complete tech spec of what dvc.lock contains? The `checksum` field, for example, is not even mentioned anywhere else in docs yet.

Reply:
I did change it to "hash value" in the paragraph though. Resolving this one.